js/views/transactions/SalesTab.js

```diff
@@ -0,0 +1,101 @@
+import app from '../../app';
+import Tab from './Tab';
+import resyncBlockchain, {
+  isResyncAvailable,
+  isResyncingBlockchain,
+  events as resyncEvents,
+} from '../../utils/resyncBlockchain';
+import ResyncPopInMessage from './ResyncPopInMessage';
+
+export default class extends Tab {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    if (!options.unfilteredSalesFetch ||
+      !(typeof options.unfilteredSalesFetch.then === 'function')) {
+      throw new Error('Please provide a promise containing the fetch of the unfiltered ' +
+        'sales list.');
+    }
+
+    options.unfilteredSalesFetch.done(() => this.checkShowResyncPopin());
+
+    this.listenTo(resyncEvents, 'changeResyncAvailable', () => {
+      if (!this.resyncPopinMessage) {
+        this.checkShowResyncPopin();
+      }
+    });
+
+    this.listenTo(resyncEvents, 'resyncing', () => {
+      if (this.resyncPopinMessage) {
+        this.resyncPopinMessage.setState({
+          isSyncing: true,
+        });
+      }
+    });
+
+    this.listenTo(resyncEvents, 'resyncComplete', () => {
+      if (this.resyncPopinMessage) {
+        this.resyncPopinMessage.setState({
+          isSyncing: false,
+          complete: true,
+        });
+      }
+    });
+
+    this.listenTo(resyncEvents, 'resyncFail', () => {
+      if (this.resyncPopinMessage) {
+        this.resyncPopinMessage.setState({
+          isSyncing: false,
+        });
+      }
+    });
+  }
+
+  get transactionAgeForResync() {
+    return 1000 * 60 * 60 * 24; // 24 hours
```

Comment:
You could muck with this if you want to see the resync pop-up on the sales tab and not have to wait for a transaction to be unconfirmed for the 24 hours listed here.